internal/utility/purl/composer_extractor.go

```diff
@@ -0,0 +1,21 @@
+package purl
+
+import (
+	"log"
+	"strings"
+
+	"github.com/google/osv-scanner/pkg/models"
+)
+
+func ExtractPURLFromComposer(packageInfo models.PackageInfo) (namespace string, name string, ok bool) {
```

Comment:
nit: `Extract` is a bit overloaded for us -- we already use it in the context of "extracting" dependencies from e.g. a lockfile. 

Could we just omit the `extract` bit in this directory? And further following [Go conventions to avoid repetition](https://google.github.io/styleguide/go/best-practices#avoid-repetition), could we just call these functions `FromComposer`, `FromMaven` etc? Then users will just be calling `purl.FromComposer`, `purl.FromMaven`.